docs/metrics/getting-started-prometheus-grafana/Program.cs

```diff
@@ -29,7 +29,12 @@ public static void Main()
     {
         using var meterProvider = Sdk.CreateMeterProviderBuilder()
             .AddMeter("MyCompany.MyProduct.MyLibrary")
-            .AddPrometheusHttpListener()
+            .AddConsoleExporter()
+            .AddOtlpExporter(options =>
+            {
+                options.Endpoint = new Uri("http://localhost:9090/api/v1/otlp/v1/metrics");
+                options.Protocol = OpenTelemetry.Exporter.OtlpExportProtocol.HttpProtobuf;
```

Comment:
Nit: A using statement above and not including the full namespace here would be a more typical style.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/4878